rust/rsc/src/rsc/main.rs

```diff
@@ -146,6 +147,22 @@ fn create_router(
                 move |req, next| api_key_check::api_key_check_middleware(req, next, conn.clone())
             })),
         )
+        .route(
+            "/blob",
+            post({
+                let conn = conn.clone();
+                let active = active_store.clone();
+                let dbonly = dbonly_store.clone();
+
+                move |multipart: Multipart| blob::create_blob(multipart, conn, active, dbonly)
+            })
+            .layer(DefaultBodyLimit::disable())
+            .layer(axum::middleware::from_fn({
+                let conn = conn.clone();
+                move |req, next| api_key_check::api_key_check_middleware(req, next, conn.clone())
```

Comment:
Is the double `conn.clone()` correct/needed?

Reply:
Tried a couple variant but yeah looks like it is needed. Fortunately the first clone drops out of scope quite quickly